modules/samples/s0/gadgets/usa-population-by-ethnicity/js/custom/donut-chart-gadget.js

```diff
@@ -507,7 +520,8 @@ var initDonutChart;
                 .attr("stroke", "white")
                 .attr("stroke-width", 0.5)
                 .attr("fill", function (d, i) {
-                    return color(i);
+                    console.log(d);
```

Comment:
Having console.log() is not recommended, you can use it in the debugging but not in the final code. Please remove this..